pyproject.toml

```diff
@@ -110,6 +111,10 @@ uuid6 = [
   {version = "^2022.10.25", optional = true, python = "<3.8"},
   {version = ">=2022.10.25", optional = true, python = ">=3.8"},
 ]
+cffi = [
+  {version = "^1.15", optional = true, python = "<3.8"},
+  {version = ">=1.17", optional = true, python = ">=3.8"},
+]
```

Comment:
Probably upgrading psycopg2-binary is enough, please check

Reply:
fails with:

```
Because no versions of cffi match >1.17,<1.17.1 || >1.17.1,<2.0
 and cffi (1.17.0) requires Python >=3.8, cffi is forbidden.
So, because cffi (1.17.1) requires Python >=3.8
 and data-horizon depends on cffi (^1.17), version solving failed.
```